extensions/cast/src/main/java/com/google/android/exoplayer2/ext/cast/CastPlayer.java

```diff
@@ -145,42 +145,57 @@ public CastPlayer(CastContext castContext) {
     pendingSeekWindowIndex = C.INDEX_UNSET;
     pendingSeekPositionMs = C.TIME_UNSET;
     updateInternalState();
+
+    if (remoteMediaClient != null) {
+      try {
```

Comment:
1. Wrong getCurrentWindowIndex after setting media sources when CastPlayer is connected to Chromecast device.